lib/screens/activity_edit_sheet.dart

```diff
@@ -66,6 +70,22 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
     setState(() {});
   }
 
+  void onDateTap() async {
+
+    dynamic date = await Navigator.of(context).push(MaterialPageRoute(
```

Comment:
add the route in rout_generator.dart and use named route.

Reply:
Done